src/components/EnvelopeSkeleton.vue

```diff
@@ -633,6 +637,15 @@ export default {
 		justify-content: center;
 		align-self: start;
 		margin-top: 0;
+		display: flex;
+		flex-direction: column-reverse;
+		align-items: flex-end;
```

Comment:
these will not be applied because the parent is display none. It still jumps for me

Reply:
@GretaD ?

Reply:
Continued at https://github.com/nextcloud/mail/pull/10583#issuecomment-2838605655